packages/react-core/src/components/Page/examples/Page.md

```diff
@@ -23,49 +23,89 @@ import './page.css';
 
 ## Examples
 
-### Vertical nav
+### Basic page components
+
+A page will typically contain the following components: 
+
+- A header [toolbar](/components/toolbar) with `<ToolbarContent>`, which often contains actions, icons, buttons, and/or dropdown menus.
+- A [masthead](/components/masthead) with a `<MastheadToggle>` that contains the `<PageToggleButton>`, a `<MastheadMain>`that contains the `<MastheadBrand>`, and `<MastheadContent>` that contains the page's header toolbar. 
+- A `<Page>` with, a `header` that often contains a masthead, a `<PageSidebar>`, and one or more `<PageSection>` components. 
+
+### Vertical navigation
+
+To add vertical side navigation to a `<Page>`, pass a `<PageSidebar>` component into the `sidebar` property. To render navigation in the `<PageSidebar>`, use the `nav` property. 
+
+The `isNavOpen` property helps facilitate the opening and closing of the sidebar and should be 'true' when the navigation sidebar is visible. 
 
 ```ts file="./PageVerticalNav.tsx"
 ```
 
-### Horizontal nav
+### Legacy vertical navigation
+
+This example shows the legacy implementation of a page's vertical navigation. Our updated recommendation advises you to use a masthead and toolbar to make headers, rather than `<PageHeader>` and `<PageHeaderTools>` as shown in the following example.
+
+```ts file="./PageVerticalNavUsingPageHeaderComponent.tsx"
+```
+
+### Horizontal navigation
+
+To add horizontal top navigation to a `<Page>`, each navigation item should be set as a `<ToolbarItem>` in the `<Toolbar>` that is passed to the `<Masthead>`.
 
 ```ts file="./PageHorizontalNav.tsx"
 ```
 
-### Tertiary nav
+### Tertiary navigation
+
+To add a tertiary navigation component to a page, use the `tertiaryNav` property. The following example passes a string of ‘Navigation’ to `tertiaryNav`, but you can pass fully built-out components, such as navigation.
+
+In this example, `<Masthead>` sets the `display` property equal to "{{ default: 'stack' }}" to stack the masthead items and also passes in "inset={{ default: 'insetXs' }}" to adjust the inset padding.
```

Comment:
This line needs to be updated to:

In this example, `<Masthead>` sets the `display` property equal to `"{{ default: 'stack' }}"` to stack the masthead items and also passes in `"inset={{ default: 'insetXs' }}"` to adjust the inset padding.

(so that there are tics around the double quotes). That will fix the build issue.